phase0/CRC Design/TimeDecision.txt

```diff
@@ -1,6 +1,7 @@
 Class name: TimeDecision (future phases)
 Parent class: entities.Decision
```

Comment:
think this should just be decision if everything else doesn't have entities in front of it